src/chain/chaininterface.rs

```diff
@@ -54,44 +60,79 @@ pub enum ConfirmationTarget {
 /// called from inside the library in response to ChainListener events, P2P events, or timer
 /// events).
 pub trait FeeEstimator: Sync + Send {
-	fn get_est_sat_per_vbyte(&self, ConfirmationTarget) -> u64;
+	fn get_est_sat_per_vbyte(&self, confirmation_target: ConfirmationTarget) -> u64;
```

Comment:
nit: Does it matter?

Reply:
compiler complained that anonymous arguments are deprecated.